dietpi/dietpi-software

```diff
@@ -8385,7 +8404,97 @@ _EOF_
 			mkdir -p /etc/exports.d
 			[[ -f '/etc/exports.d/dietpi.exports' ]] || echo '/mnt/dietpi_userdata *(rw,async,no_root_squash,fsid=0,crossmnt,no_subtree_check)' > /etc/exports.d/dietpi.exports
 
-		fi
+		fi 
+    
+    software_id=137
+    if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+    
+      Banner_Configuration
+
+      # Download list of root hints
+      Download_Install https://www.internic.net/domain/named.root /var/lib/unbound/root.hints
+
+      cat << '_EOF_' > /etc/unbound/unbound.conf.d/dietpi.conf
+server:
+    verbosity: 0
+    
+    interface: 0.0.0.0
+    port: 53
+    do-ip4: yes
+    do-udp: yes
+    do-tcp: yes
+    
+    do-ip6: no
```

Comment:
Any reason to not allow IPv6? Should be rare in local network but at least you explicitly allow local IPv6 addresses below. Best would be actually to set those two based on the `CONFIG_ENABLE_IPV6` and `CONFIG_PREFER_IPV4` options in `dietpi.txt`. However, I aim to remove the second soon, since it cannot be applied system-wide, so behaves inconsistent, means either IPv6 is enabled and then preferred by applications (usually) or it is disabled completely if it does not work well.